ryu/services/protocols/ovsdb/client.py

```diff
@@ -306,6 +307,16 @@ def modify_request_handler(self, ev):
         txn._uuidize()
         self._txn_q.append((txn, ev))
 
+    def read_request_handler(self, ev):
+        table = {}
+        if ev.table_name in self._idl.tables:
+            for row_uuid, row in \
+                    self._idl.tables[ev.table_name].rows.iteritems():
```

Comment:
Backslash line continuations are generally frowned upon in python and especially in openstack/ryu.  Here's a way to get around it:

```python
def read_request_hander(self, ev):
    table = None
    if ev.table_name in self._idl.tables:
        rows = self._idl.tables[ev.table_name].rows
        table = dict((row_uuid, dictify(row))
                     for row_uuid, row in rows.iteritems())
   rep = event.EventReadReply(self.system_id, table)
   self.reply_to_request(ev, rep)
```

This also has the benifit of being able to tell if a table doesn't exist (since it will return None) vs an empty table.